roles/common/defaults/main.yml

```diff
@@ -9,20 +9,17 @@ rhsm_repos: []
 kerberos_realm: EXAMPLE.COM
 
 epel_mirror_baseurl: "http://dl.fedoraproject.org/pub/epel"
-epel_mirrorlist: "http://mirrors.fedoraproject.org"
 epel_repos:
   epel:
     name: "Extra Packages for Enterprise Linux"
-    baseurl: "{{ epel_mirror_baseurl }}/{{ ansible_distribution_major_version }}/$basearch"
-    mirrorlist: "{{ epel_mirrorlist }}/metalink?repo=epel-{{ ansible_distribution_major_version }}&arch=$basearch"
+    baseurl: "{{ epel_mirror_baseurl }}/{{ ansible_distribution_major_version }}/$basearch\nhttp://mirrors.cat.pdx.edu/epel/{{ ansible_distribution_major_version }}/$basearch\nhttp://mirror.pnl.gov/epel/{{ ansible_distribution_major_version }}/$basearch\nhttp://fedora-epel.mirror.lstn.net/{{ ansible_distribution_major_version }}/$basearch"
```

Comment:
A template might be useful here:
http://docs.ansible.com/ansible/template_module.html